graylog2-server/src/main/java/org/graylog2/alarmcallbacks/EmailAlarmCallback.java

```diff
@@ -85,7 +74,7 @@ public void call(Stream stream, AlertCondition.CheckResult result) {
                         .addType(Notification.Type.EMAIL_TRANSPORT_CONFIGURATION_INVALID)
                         .addSeverity(Notification.Severity.NORMAL)
                         .addDetail("stream_id", stream.getId())
-                        .addDetail("exception", Throwables.getStackTraceAsString(e));
+                        .addDetail("exception", e.getMessage());
```

Comment:
Just saw that `TransportConfigurationException` doesn't take a root cause exception in its constructor, so this is actually fine.